### DISCUSSION TITLE
hi. couple of questions! :)

### DISCUSSION THREAD

**Discussion by omgkhaleesi** [drizzle-mizzle/CharacterAI-Discord-Bot#143] [General]
hello! loving my bot(s) and it works perfectly in my discord. we're having a lot of fun. 
couple of questions. 

1. I've created two seperate character ai bots and they are both in the same discord. I make them talk to each other, if I make them stop (using the ⛔ reaction), they wont talk again in the same channel when I use the @character-name #call @othercharacter-name (text) for the second time. Is there a way to prevent this? 

2. Do I have to #find and choose the character AI every time I load up? This is what Ive been doing and Im not sure if I either did something wrong or its just normal 

3. Might be a stupid question, but do I essentially have to leave my computer on to keep these bots running? People love talking to them on my discord but if I go to sleep without leaving my computer on they cant. or// Do I have to keep the terminal open for them to work? When I close them, the bots go offline and I have to #find and choose the character again.

**Comment by drizzle-mizzle** [comment]
> if I make them stop (using the ⛔ reaction), they wont talk again in the same channel when I use the https://github.com/character-name #call @othercharacter-name (text) for the second time. Is there a way to prevent this?

I'm sure it should respond.
This button simply sets "ignore next bot message" flag to true, and when it gets a message from any bot, it ignores it and then sets it to false, so the following messages from bots are not ignored.
Maybe you have pressed it for both bots? Then the second one will ignore one message too. Try to use `call` again.

> Do I have to #find and choose the character AI every time I load up?

[Auto setup](https://github.com/drizzle-mizzle/CharacterAI-Discord-Bot/wiki/Additional-configuration#set-auto_setup-to-true-and-specify-auto_char_id-with-id-of-a-character-if-you-want-the-bot-to-set-character-automatically-after-every-relaunch)

> Might be a stupid question, but do I essentially have to leave my computer on to keep these bots running?

You do, or you could [get some cloud server](https://www.google.com/search?q=free+vps).

**Reply by omgkhaleesi** [reply]
Thank you for the quick reply! First two worked, easy fix made me feel silly. Gonna check out the cloud server. Thanks again.
